src/mygrad/linalg/funcs.py

```diff
@@ -113,14 +118,25 @@ def norm(
     >>> l2_norms
     Tensor([3.74165739, 1.        ])
 
-    The presence of the elementwise absolute values in the norm means that zero-valued
-    entries in a vectors have an undefined derivative.
+    When `nan_to_num=False` is specified the presence of the elementwise absolute values in the
+    norm operation means that zero-valued entries in any of input vectors will have an undefined derivative.
```

Comment:
```suggestion
    The presence of the elementwise absolute values in the norm operation means that zero-valued entries in any of 
    input vectors have an undefined derivative. When `nan_to_num=False` is specified these derivatives will be reported
    as `nan`, otherwise they will be made to be 0.0.
```